jobserver/urls.py

```diff
@@ -255,6 +255,7 @@
     path("admin/approve-users", ApproveUsers.as_view(), name="approve-users"),
     path("admin/", admin.site.urls),
     path("api/v2/", include((api_urls, "api"))),
+    path("applications/", include("applications.urls")),
```

Comment:
If you copy the namespace bit of the`staff` include, we can reference urls with the namespace, eg `"applications:some_name"`, while only doing `name="some_name"` in the urls.py

Reply:
How's this different to having `app_name = "applications"` in `applications/urls.py`?

Reply:
I thought you had to have both… do you not??  Useful to see the name explicitly used as a namespace though I think?

Reply:
This is a complicated corner of Django that's new to me.

Pretty sure you don't have to have both -- [from the docs](https://docs.djangoproject.com/en/dev/topics/http/urls/#reversing-namespaced-urls) I think that the "namespace=" argument to `path()` (the "instance namespace") is only used if you have multiple instances of the app (at different URLs).

I can see the appeal of having the namespace in two places, but I'd rather not, since you might end up with a confusing situation if you ever change one of them.

Reply:
I didn't realise you could just use the app_name!  I'd have prefered the namespace to be explicit in the main urls.py but I agree on not having it in two places 👍